checkov/terraform/checks/resource/aws/ElasticsearchDomainLogging.py

```diff
@@ -8,13 +8,24 @@ class ElasticsearchDomainLogging(BaseResourceValueCheck):
     def __init__(self):
         name = "Ensure Elasticsearch Domain Logging is enabled"
         id = "CKV_AWS_84"
-        supported_resources = ['aws_elasticsearch_domain']
+        supported_resources = ['aws_elasticsearch_domain', 'aws_opensearch_domain']
         categories = [CheckCategories.LOGGING]
         super().__init__(name=name, id=id, categories=categories, supported_resources=supported_resources)
 
     def get_inspected_key(self):
         return "log_publishing_options/[0]/cloudwatch_log_group_arn"
 
+    def scan_resource_conf(self, conf):
+        if conf.get("log_publishing_options") and isinstance(conf.get("log_publishing_options"), list):
+            option = conf.get("log_publishing_options")[0]
+            if option.get('cloudwatch_log_group_arn'):
```

Comment:
```suggestion
            if isinstance(option, dict) and option.get('cloudwatch_log_group_arn'):
```
I think that's the issue, but you need to test it 😄